### DISCUSSION TITLE
Lets Shorten the Title of Compound Tiddler in Type Field

### DISCUSSION THREAD

**Discussion by kookma** [TiddlyWiki/TiddlyWiki5#8465] [Ideas]
I think @pmario has updated recently the Type field drop down and it is quite better than before. One small suggestion is to shorten the title from

`TiddlyWiki 5 Compound Tiddler (text/vnd.tiddlywiki-multiple)`

to

`Compound Tiddler (text/vnd.tiddlywiki-multiple)`

![image](https://github.com/user-attachments/assets/204f83ac-a789-42c6-978c-099761f8980a)

* I think Compound Tiddler is clear that it is related to TiddlyWiki
* The `Compound Tiddler` is much similar to other types under `Developer`
* The drop down menu width is very long now specially on mobile, this way it will be shorter

**Comment by pmario** [comment]
What's the problem with the longer text? I did try it on my mobile phone and it works well there too.

**Reply by kookma** [reply]
Well in my cell phone 5.2", I had to scroll.
This is not a bug only a design point! Thank you for all your efforts.

**Comment by Jermolene** [comment]
Thanks @kookma I agree, and would welcome a PR with this change

**Reply by kookma** [reply]
Thank you @Jermolene. The PR was submitted: See #8469.
